### ISSUE TITLE
Personal Access Token (PAT) no longer available as a key type

### ISSUE THREAD

**Issue by bkraul** [ansible-semaphore/semaphore#1489]
I just updated my semaphore setup to 2.9.19, and I find that PAT is no longer a key type. I see that the login with password now has the login as optional. However, I tried to set the token in the password field, expecting the key to work the same as a PAT, and it does not seem to be working as such. The task just stays on "Waiting" status.

What is the course of action here?

I am seeing on the documentation there is still a Personal Access Token section but it is empty.

![image](https://github.com/ansible-semaphore/semaphore/assets/14616851/7fd19a9b-6e49-4c04-acb9-ebacb6305bda)

**Comment by fiftin** [comment]
Hi @bkraul 

It is not required because you can use `Login with Password` instead of it. I will update docs. Thank you!

**Comment by bkraul** [comment]
Yeah I saw that in one of the release notes. However, the Waiting status really is an issue. I went back all the way to version 2.8.53, and verified the task works...but when I update to latest (now 2.9.21) it just sits there with "Waiting" status. I have tried to check for any logs that can give me any information as to what's happening, and I get nothing.

The log only says: `Added task [XX]`

**Comment by fiftin** [comment]
Hi @bkraul I already fixed the issue. Please check latest pre-release.
